modules/virt-cnv-network-alerts.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-events.html/virt-virtualization-alerts.adoc
+
+[id="virt-cnv-network-alerts_{context}"]
+= Network alerts
+
+Network alerts provide information about problems for the CNV Network Operator.
+
+//KubeMacPoolDown Alert
+[id="KubeMacPoolDown_{context}"]
+== KubeMacPoolDown alert
+
+.Description
+
+The KubeMacPool is responsible for allocating MAC Addresses and preventing MAC Address conflicts.
+
+.Reason
+
+If the KubeMacPool-manager pod is down, then creating `VirtualMachine` objects fails.
+
+.Troubleshoot
+
+* Determine the Kubemacpool-manager’s pod namespace and name.
++
+[source,terminal]
+----
+export KMP_NAMESPACE="$(oc get pod -A --no-headers -l control-plane=mac-controller-manager | awk '{print $1}')"
+
+export KMP_NAME="$(oc get pod -A --no-headers -l control-plane=mac-controller-manager | awk '{print $2}')"
+----
+
+* Check the Kubemacpool-manager pod's pod description and logs.
++
+[source,terminal]
+----
+oc describe pod -n $KMP_NAMESPACE $KMP_NAME
+
+oc logs -n $KMP_NAMESPACE $KMP_NAME
+----
+
```

Comment:
And - look for what? An entry of `Alerts`? 
An output example might be useful.

Reply:
@RamLavi Can you please help here?

Reply:
>@RamLavi Would it help to add something in the Reason section saying "The KubeMacPool-manager pod may be down for several reasons, such as" and listing a few of the possible reasons?

That's a strong no on that . 
Only reason for this alert to go on is if there's a bug in kubemacpool. kubemacpoolDown alert is relevant because of the severity of the outcome when down (meaning, no vms can be created/updated), but it isn't to say that it is more prone to be down than any other pod.

>Or perhaps in this oc logs step, we could say "Check the Kubemacpool-manager pod description and logs to determine the source of the problem."?

That is better, I would vote for that. I would also address the sysAdmin to open a case about it, as it is probably not something they can fix.